control-plane/typeSpecConverter.cpp

```diff
@@ -111,6 +115,48 @@ bool TypeSpecConverter::preorder(const IR::Type_Name* type) {
     return false;
 }
 
+// This function is invoked if psa.p4 has a Newtype. The function
```

Comment:
This code has nothing to do with PSA, so there should be no comments related to PSA in it. This file applies to all architectures.

There is no notion of user-defined NewType. I don't know what this comment means.

Reply:
There is `type` in a model file such as p4c/p4include/psa.p4.  This method is able to process only `type` in psa.p4.  If a user writes a P4 program and uses `type`, this is what I call as user-defined NewType.  A user-defined NewType is not handled by this method.